tsdb/db.go

```diff
@@ -1499,9 +1499,13 @@ func (db *DB) CleanTombstones() (err error) {
 			return err
 		} else if uid != nil { // New block was created.
 			newUIDs = append(newUIDs, *uid)
+			if err = errors.Wrap(db.reload(), "reload blocks"); err != nil {
+				return err
+			}
```

Comment:
To be consistent with rest of the code
```suggestion
			if err = db.reload(); err != nil {
				return errors.Wrap(err, "reload blocks")
			}
```